_data/nav.yml

```diff
@@ -0,0 +1,10 @@
+head:
+    - Home
+    - title: Install
+      url: /install
+    - About Us
+    - Documentation
+    - title: News
+      url: /news
+    - title: Releases
+      url: /releases
```

Comment:
We could add the code of conduct as a header `conduct` and a separate page, now or in future. I think it is hard to find at the moment.

Reply:
How about a new page `community` and on that page we cover coc, mailing lists, getting involved links etc?